src/components/contacts/screens/PhoneInput.tsx

```diff
@@ -0,0 +1,286 @@
+              />
+            </View>
+            <View style={[a.flex_1]}>
+              <TextField.Root isInvalid={!!formatError || !isFeatureEnabled}>
+                <TextField.Input
+                  label={_(msg`Phone number`)}
+                  value={phoneNumber}
+                  onChangeText={text => {
+                    if (formatError) setFormatError('')
+                    if (Math.abs(text.length - phoneNumber.length) > 1) {
+                      // possibly pasted/autocompleted? auto-switch
+                      // country code if possible
+                      const result = getCountryCodeFromPastedNumber(text)
+                      if (result) {
+                        setCountryCode(result.countryCode)
+                        setPhoneNumber(result.rest)
+                        return
+                      }
+                    }
+                    setPhoneNumber(text)
+                  }}
+                  placeholder={null}
+                  keyboardType="number-pad" // we don't want people entering +() etc
+                  autoComplete="tel"
+                  returnKeyType={android('next')}
+                  onSubmitEditing={onSubmitNumber}
+                />
+              </TextField.Root>
+            </View>
+          </View>
+        </View>
+
+        {!isFeatureEnabled && (
+          <ErrorText>
+            <Trans>
+              Support for this feature in your country has not been enabled yet!
+              Please check back later.
+            </Trans>
+          </ErrorText>
+        )}
+        {error && <ErrorText>{error}</ErrorText>}
+        {formatError && <ErrorText>{formatError}</ErrorText>}
+
+        <View style={[a.mt_auto, a.py_xl]}>
+          <LegalDisclaimer />
+        </View>
+      </Layout.Content>
+      <KeyboardAvoidingView
+        behavior="padding"
+        keyboardVerticalOffset={insets.top - paddingBottom + tokens.space.xl}>
+        <View style={[gutters, {paddingBottom}]}>
+          <Button
+            disabled={!phoneNumber || isPending}
+            label={_(msg`Send code`)}
+            size="large"
+            color="primary"
+            onPress={onSubmitNumber}>
+            <ButtonText>
+              <Trans>Send code</Trans>
+            </ButtonText>
+            {isPending && <ButtonIcon icon={Loader} />}
+          </Button>
+        </View>
+      </KeyboardAvoidingView>
+    </View>
+  )
+}
+
+function LegalDisclaimer() {
+  const t = useTheme()
+  const {_} = useLingui()
+
+  const style = [a.text_xs, t.atoms.text_contrast_medium, a.leading_snug]
+
+  return (
+    <View style={[a.gap_xs]}>
+      <Text style={[style, a.font_medium]}>
+        <Trans>How we use your number:</Trans>
+      </Text>
+      <Text style={style}>
+        &bull;{' '}
+        <Trans>Sent to our phone number verification provider Plivo</Trans>
+      </Text>
+      <Text style={style}>
+        &bull; <Trans>Deleted by the verifier after verification</Trans>
```

Comment:
```suggestion
        &bull; <Trans>Deleted by Plivo after verification</Trans>
```
nit: as Plivo has already been mentioned on the preceding line, suggest referring to it here rather than `the verifier`